src/librawspeed/decompressors/FujiDecompressor.cpp

```diff
@@ -0,0 +1,1097 @@
+/*
+    FujiDecompressor - Decompress Fujifilm compressed RAF.
+
+    Copyright (C) 2016 Alexey Danilchenko
+    Copyright (C) 2016 Alex Tutubalin
+    Copyright (C) 2017 Uwe Müssel
+
+    This library is free software; you can redistribute it and/or
+    modify it under the terms of the GNU Lesser General Public
+    License as published by the Free Software Foundation; either
+    version 2 of the License, or (at your option) any later version.
```

Comment:
Is it enough to change the version from 2 to 2.1 or do I have to copy the whole LibRaw license header?